src/blurredLocationDisplay.js

```diff
@@ -151,6 +151,7 @@ BlurredLocationDisplay = function BlurredLocationDisplay(options) {
 
               var id = obj["id"] ;
               var url = obj["url"] ;
+              var title = obj["title"] ;
```

Comment:
Here, it should work, i think:

https://github.com/publiclab/leaflet-blurred-location-display/blob/86e873954769cc01ef8061e3bfa9237f09276c87/src/blurredLocationDisplay.js#L60